sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcOptions.scala

```diff
@@ -67,4 +67,11 @@ object OrcOptions {
     "snappy" -> "SNAPPY",
     "zlib" -> "ZLIB",
     "lzo" -> "LZO")
+
+  // The extensions for ORC compression codecs
+  val extensionsForCompressionCodecNames = Map(
```

Comment:
this doesn't belong to `OrcOptions`, maybe `OrcUtils`?

Reply:
It's moved to OrcUtils.